src/main/groovy/com/stormpath/tck/util/EnvUtils.groovy

```diff
@@ -22,16 +22,19 @@ class EnvUtils {
 
     public static final String stormpathHtmlEnabled = getVal("STORMPATH_TCK_HTML_ENABLED", "true")
 
-    public static final String jwtSigningKey
     public static final String facebookClientId
     public static final String facebookClientSecret
 
+    public static final String jwtSigningKeyModulus
+    public static final String jwtSigningKeyExponent
+
     static {
-        jwtSigningKey = getVal("JWT_SIGNING_KEY")
+        jwtSigningKeyModulus = getVal("JWT_SIGNING_KEY_MOD")
+        jwtSigningKeyExponent = getVal("JWT_SIGNING_KEY_EXP")
         facebookClientId = getVal("FACEBOOK_CLIENT_ID")
         facebookClientSecret = getVal("FACEBOOK_CLIENT_SECRET")
-        if (jwtSigningKey == null || facebookClientId == null || facebookClientSecret == null) {
-            fail("JWT_SIGNING_KEY, FACEBOOK_CLIENT_ID and FACEBOOK_CLIENT_SECRET environment variables are required")
+        if (jwtSigningKeyModulus == null || jwtSigningKeyExponent == null || facebookClientId == null || facebookClientSecret == null) {
```

Comment:
we should probably allow setting both. Or we could just set the `keys` endpoint and download the keys as needed?